### PR TITLE
dx-1242 Fixed the PATCH code example

### PR DESCRIPTION
**Pull request by arebra**

### CODE REVIEW

File: _includes/delete-token.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ No other states are supported.
 **Request**
 
 ```http
-PATCH /psp/{{ api_resource }}/{% unless api_resource == "paymentorders" %}payments/instrumentdata{% else %}paymentorders/recurrenceTokens{% endunless %}{{ page.payment_token }} HTTP/1.1 
+PATCH /psp/{{ api_resource }}/{% if api_resource == “paymentorders” %}recurrenceTokens/{{ page.payment_token }} HTTP/1.1 {% else %}payments/instrumentData/{{ page.payment_token }} HTTP/1.1 {% endif %}
```
Comment (asbjornu) [inline _includes/delete-token.md:14]:
```suggestion
PATCH /psp/{{ api_resource }}/{% if api_resource == "paymentorders" %}recurrenceTokens/{{ page.payment_token }} HTTP/1.1 {% else %}payments/instrumentData/{{ page.payment_token }} HTTP/1.1 {% endif %}
```

File: _includes/delete-token.md [hunk 2]
```diff
@@ -11,14 +11,14 @@ No other states are supported.
...
 Host: {{ page.api_host }}
 Authorization: Bearer <AccessToken>
 Content-Type: application/json
 {
-  "state": "Deleted",
-  "comment": "Comment on why the deletion is happening"{% if documentation_section == "card" %},
-  "tokenType" : "{{ token_field_name }}"{% endif %}
+  “state”: “Deleted”,
+  “comment”: “Comment on why the deletion is happening”{% if documentation_section == “card” %},
+  “tokenType” : “{{ token_field_name }}“{% endif %}
```
Comment (asbjornu) [inline _includes/delete-token.md:21]:
```suggestion
  "state": "Deleted",
  "comment": "Comment on why the deletion is happening"{% if documentation_section == "card" %},
  "tokenType" : "{{ token_field_name }}"{% endif %}
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
I think it's wise to avoid “smart quotes” in code.

File: _includes/delete-token.md [hunk 3]
```diff
@@ -11,14 +11,14 @@ No other states are supported.
 **Request**
```
Comment (asbjornu) [inline _includes/delete-token.md:12]:
```suggestion
{% capture patch_url %}
    /psp/{{ api_resource }}/
    {% if api_resource == "paymentorders" %}
        recurrenceTokens
    {% else %}
        payments/instrumentData
    {% endif %}
    /{{ page.payment_token }}
{% endcapture %}
{% assign patch_url=patch_url | strip_newlines %}

```

**Review by asbjornu — CHANGES_REQUESTED** [review]
The URLl building becomes easier to understand if we do it outside of the code example.

**Review by asbjornu — CHANGES_REQUESTED** [review]
I made a few changes. Please take a peek at 1dd543858c79d10b2a7d26673d79881ff5535653 where I describe the changes I made and why.

Left to do now is to ensure that `delete-token.md` is only included once in Other Features. I think it's repeated 3 times underneath Card, for instance. Its inclusion in `card-purchase.md` doesn't seem to make much sense, for instance.

**Comment by asbjornu** [comment]
https://github.com/SwedbankPay/developer.swedbankpay.com/pull/708#pullrequestreview-506601232 still needs some followup, no?

**Comment by arebra** [comment]
Is there anything I can do with #708? I can't approve the review of my own task? Or am I missing something?

**Comment by asbjornu** [comment]
I still can't see that the following has been done? The last commit in this PR was made by me, so I don't understand where you've removed the include duplicates, @arebra?

> Left to do now is to ensure that `delete-token.md` is only included once in Other Features. I think it's repeated 3 times underneath Card, for instance. Its inclusion in `card-purchase.md` doesn't seem to make much sense, for instance.

**Comment by arebra** [comment]
That's odd, but I guess I didn't commit and push it correctly then. I'll fix it :)

**Review by asbjornu — APPROVED** [review]

**Pull request merged**